lucene/analysis/common/src/java/org/apache/lucene/analysis/hunspell/Dictionary.java

```diff
@@ -70,6 +70,9 @@
 
 /** In-memory structure for the dictionary (.dic) and affix (.aff) data of a hunspell dictionary. */
 public class Dictionary {
+  // Derived from woorm/ openoffice dictionaries.
```

Comment:
LibreOffice?

Reply:
Ouch. Can you correct it for my and piggyback on any other subsequent patch? I overlooked this one.